.gitignore

```diff
@@ -96,3 +100,10 @@ dump.rdb
 
 # Visual Studio Code
 .vscode
+.idea
+
+# devops files
+bin/terrafile
+
+# Downloaded terraform modules
+terraform/aks/vendor/
```

Comment:
duplicate entry, although the other one is terraform/aks/vendor without the ending / 
so just one or the other